tests/external/search_packages_test.php

```diff
@@ -1033,5 +1022,145 @@ public function test_mine_with_context_version_and_server_version_are_both_retur
         $this->assertEqualsCanonicalizing(['0.1.0', '0.2.0'], array_column($versions, 'version'));
     }
 
+    /**
+     * Favours one or multiple packages given their id(s).
+     *
+     * @param user_favourite_service $ufservice
+     * @param context_user $context
+     * @param int ...$ids the package version ids to be favoured
+     * @throws moodle_exception
+     */
+    private static function favour(user_favourite_service $ufservice, context_user $context, int ...$ids): void {
```

Comment:
"to favo(u)r" heißt sowas wie "bevorzugen". "zum Favoriten machen" ist eher "to favo(u)rite". Dass auf deutsch beides "favorisieren" heißt, wusste ich noch nicht :see_no_evil: 

```suggestion
    /**
     * Favourites one or multiple packages given their id(s).
     *
     * @param user_favourite_service $ufservice
     * @param context_user $context
     * @param int ...$ids the package version ids to be favourited
     * @throws moodle_exception
     */
    private static function favourite(user_favourite_service $ufservice, context_user $context, int ...$ids): void {
```

Auch an den anderen Stellen